gen/dcos-config.yaml

```diff
@@ -90,9 +90,91 @@ package:
           ]
         }
       ].
+  - path: /etc_master/navstar.app.config
```

Comment:
Crap. Looks like it got pulled in from my earlier cherry-picks. Let me remove it from the commit that bought it in.